src/utils/index.js

```diff
@@ -72,6 +76,26 @@ export default {
     }
   },
 
+  updateFieldGroup (columns) {
+    const allColumns = [].concat(...columns)
```

Comment:
conact a empty array ?

Reply:
Because the columns is a two-dimensional array, so here need to convert it to one dimension, and then we can use the `find` method.
We can use `[].concat` method to do this (ES6), for example:
```
const arr = [[1], [3]]
[].concat(...arr)
// result: [1, 3]
```

Reply:
I found that we can use `flat` function instead of `contact`, and it's better to understand.